website/astro.config.ts

```diff
@@ -1,40 +1,9 @@
-import { netlifyStatic } from "@astrojs/netlify";
+import netlify from "@astrojs/netlify";
 import react from "@astrojs/react";
 import starlight from "@astrojs/starlight";
 import { defineConfig } from "astro/config";
-import { h } from "hastscript";
-import { escape as htmlEscape } from "html-escaper";
-import rehypeAutolinkHeadings from "rehype-autolink-headings";
+import { rehypeAutolink } from "./plugins/rehype-autolink";
 import rehypeSlug from "rehype-slug";
-import remarkToc from "remark-toc";
```

Comment:
Looks like we weren't using `"remark-toc"` anywhere, so I've removed it. Also, did the same for `@astrojs/vercel`, which isn't being used anymore.